python-heasoft-osa/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export HOME_OVERRRIDE=/tmp/jovyan; 
+
+mkdir $HOME_OVERRRIDE; 
+
+cd /tmp/jovyan; 
+
+source /init.sh; 
+
+jupyter lab --ip 0.0.0.0 --no-browser'
```

Comment:
Please allow the entrypoint to execute the command passed by the runtime, e.g. as in https://github.com/SwissDataScienceCenter/renkulab-docker/blob/main/docker/py/entrypoint.sh#L59-L60